assets/js/app.js

```diff
@@ -264,3 +264,37 @@ function setupDatePicker() {
     $(this).flatpickr(options);
   });
 }
+
+$(".password-pair").on("change", function (event) {
```

Comment:
Do we really need to verify on the frontend with JavaScript? It's already validated on the backend. If the validation fails, it returns an error along with the old inputs.

I don't see the necessity to validate the password twice. Plus, we lack a mechanism to test JavaScript code. I favor server-side validation.

<img width="2229" alt="Screenshot 2023-12-06 at 17 34 08" src="https://github.com/ubicloud/ubicloud/assets/993199/96f0ba0e-bbb1-4e66-9675-e0fe077cbe63">

Reply:
Yes, but then you only get the error message after you submit the form. Server-side validation is for us to ensure incoming data is valid, client-side validation is for user to have better experience.

Benefit of client side validation is less than usual as the form comes back already filled up. So it might be OK to remove it completely. I'm not sure. Do you think client side validation is not adding any value?

Reply:
> Do you think client side validation is not adding any value?

In my opinion, the added code does not provide value considering its maintenance cost.

> which is more serious when only server-side validation is used. The error message is only shown next to the input box, which might not be visible initially after the page is loaded (like in this case).

Also we can add an error message to the top with `flash["error"] = "my error"`.

We might add a feature to scroll down to the form adding `#` fragment to the form and returned back URL.

Reply:
I don't feel that strongly to keep the client side validation. I removed it.